SpatialGDK/Source/SpatialGDKTests/SpatialGDK/LoadBalancing/LayeredLBStrategy/LayeredLBStrategyTest.cpp

```diff
@@ -46,6 +47,18 @@ UWorld* GetAnyGameWorld()
 
 } // anonymous namespace
 
+// Disable local deployments from starting and open map
+void SetUp()
+{
+	const auto Settings = GetMutableDefault<USpatialGDKEditorSettings>();
+	const auto OldBAutoStartLocalDeployment = Settings->bAutoStartLocalDeployment;
+	Settings->bAutoStartLocalDeployment = false;
+
+	AutomationOpenMap(SpatialConstants::EMPTY_TEST_MAP_PATH);
+
+	Settings->bAutoStartLocalDeployment = OldBAutoStartLocalDeployment;
```

Comment:
ah nice. so as soon as we open the map, we can reset the value to its default? is `AutomationOpenMap` guaranteed to block until the map is open ?

Reply:
`AutomationOpenMap` doesn't block itself, just triggers the start of a deployment (if enabled)
The tests then `FWaitForWorld` once the data is prepared